src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -168,6 +169,10 @@ export default {
 			type: Boolean,
 			default: false
 		},
+		trackingCategory: {
+			type: String,
+			default: 'search'
```

Comment:
I'd recommend removing the default and instead making this a required prop. It looks like this changeset already sets this prop, so it might make sense to just make it required (also prevents the wrong category maybe being logged in the future if this component is reused).